### PR TITLE
Add support for Intel X710 Backplane and Base T

### LINKED ISSUE #453: Add support for Intel X710 Backplane and Base T
The SRIOV network operator needs support for Intel X710 10G Backplane (8086 1581) and Intel X710 10 G Base T (8086 15FF)

### PR DESCRIPTION
**Pull request by vrindle**
Closes #453

### CODE REVIEW

**Comment by wizhaoredhat** [comment]
Please follow: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/supported-hardware.md#initial-support

File: doc/supported-hardware.md [hunk 1]
```diff
@@ -42,6 +44,8 @@ The following table depicts the supported SR-IOV hardware features of each suppo
 | Model                    | SR-IOV Kernel | SR-IOV DPDK | SR-IOV Hardware Offload (switchdev) |
 | ------------------------ | ------------- | ----------- |------------------------------------ |
 | Intel XXV710 Family | V | V | X |
+| Intel X710 10GbE Backplane | 8086 | 1581 | V | V | X |
+| Intel X710 10Gbe Base T | 8086 | 15ff | V | V | X |
 | Intel X550 Family | V | V | X |
```
Comment (Eoghan1232) [inline doc/supported-hardware.md:49]:
```suggestion
| Intel X710 10GbE Backplane | V | V | X |
| Intel X710 10Gbe Base T | V | V | X |
| Intel X550 Family | V | V | X |
```

please remove the vendor and device ID from this section.

Comment (Eoghan1232) [reply doc/supported-hardware.md:49]: also please change both to ``GbE``

**Comment by Eoghan1232** [comment]
Intel X710 10Gbe Base T
 I valdiated this NIC, however I do not have the other NIC to test.

I assume these were validated @vrindle as you are currently using them :) 

Can you address the comments above and we can get this merged for you :)

**Comment by vrindle** [comment]
> Intel X710 10Gbe Base T I valdiated this NIC, however I do not have the other NIC to test.
> 
> I assume these were validated @vrindle as you are currently using them :)
> 
> Can you address the comments above and we can get this merged for you :)

I validated both of these NIC just yesterday using this guide: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/supported-hardware.md#initial-support. The comments are addressed.

**Comment by Eoghan1232** [comment]
> > Intel X710 10Gbe Base T I valdiated this NIC, however I do not have the other NIC to test.
> > I assume these were validated @vrindle as you are currently using them :)
> > Can you address the comments above and we can get this merged for you :)
> 
> I validated both of these NIC just yesterday using this guide: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/supported-hardware.md#initial-support. The comments are addressed.

Hi @vrindle , you need to add this to the Supported features per hardware of the doc/supported-hardware
```
| Intel X710 10GbE Backplane | V | V | X |
| Intel X710 10Gbe Base T    | V | V | X |
```

I think you removed it by mistake as you had it in already :)

**Comment by vrindle** [comment]
> | Intel X710 10GbE Backplane | V | V | X |

Addressed

**Review by Eoghan1232 — APPROVED** [review]
lgtm! thanks for fixing those :)

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**